configure.ac

```diff
@@ -33,15 +33,17 @@ AC_ARG_ENABLE([rdma],
 			      [enable rdma transport - default:yes ])],
 			       [enable_rdma=$enableval],
 			       [enable_rdma=yes])
-AC_MSG_RESULT([$enable_rdma])
 
 # Checks for header files.
 if test "$enable_rdma" = "yes"; then
 	AC_CHECK_HEADERS([infiniband/verbs.h rdma/rdma_cma.h],
-			 [mypj_found_verbs_headers=yes; break;])
+			 [mypj_found_verbs_headers=yes; break;],
+			 [AC_MSG_WARN([verbs headers not found, rdma disabled]);
```

Comment:
would be better to stop the build here and not modify user flags under the hood. users use automatic build tools that will not detect the change and eventually build invalid package.